app/Http/Requests/ArticleRequest.php

```diff
@@ -17,6 +17,7 @@ public function rules()
             'tags.*' => 'exists:tags,id',
             'original_url' => 'url|nullable',
             'series' => ['nullable', new AuthorOwnsSeriesRule],
+            'status' => ['required', 'in:draft,publish'],
```

Comment:
Instead of working with named statuses, make this a boolean. You can work with "1" and "0" values in the button values.

```suggestion
            'published' => ['required', 'boolean'],
```